app/src/views/internships/index.vue

```diff
@@ -136,17 +138,18 @@
 import { Component, Vue } from 'vue-property-decorator';
 import { Form } from 'element-ui';
 import { cloneDeep } from 'lodash';
-
 import {
-  getBusinesses,
-  createBusiness,
-  updateBusiness,
-  defaultBusinessData,
-} from '../../api/businesses';
-import { IBusiness } from '../../api/types';
+  getInternships,
+  createInternship,
+  updateInternship,
+  deleteInternship,
+  defaultInternshipData,
+} from '../../api/internships';
+import { IInternship } from '../../api/types';
 import { exportJson2Excel } from '../../utils/excel';
 import { formatJson } from '../../utils';
 import Pagination from '../../components/Pagination/index.vue';
+import { ID } from '../../../../services/src/api/validators/generic.val';
```

Comment:
The definition import of the back-end can not be kept in the front-end part to maintain independence between the two projects.

Must be removed